ptclient/cts/core/crud/rowstatus.js

```diff
@@ -368,9 +374,6 @@ class rowStatus extends Model {
     })
 
     await Promise.all(promises)
```

Comment:
there is no documentation saying 

Why is it is a problem if all are removed at one time?